fuzz/fuzz_targets/refcounted_model.rs

```diff
@@ -41,28 +46,85 @@ impl DbSimulator for Simulator {
 			stats: false,
 			salt: None,
 			compression_threshold: HashMap::new(),
+			always_flush: true,
+			with_background_thread: false,
 		}
 	}
 
-	fn apply_operation_on_model(operation: &Operation, model: &mut Self::Model) {
-		match *operation {
-			Operation::Set(k) => {
-				*model.entry(k).or_default() += 1;
-			},
-			Operation::Dereference(k) =>
-				if let Entry::Occupied(mut e) = model.entry(k) {
-					let counter = e.get_mut();
-					*counter = counter.saturating_sub(1);
+	fn apply_operations_on_model<'a>(
+		operations: impl IntoIterator<Item = &'a Operation>,
+		model: &mut Model,
+	) {
+		let mut counts = [None; 256];
```

Comment:
It's 256 and not 255 (=`u8::MAX`) because it's the number of possible values in a `u8`. I might do `1 << u8::BITS` but I am not sure if it's more readable.